src/interactive_creative_file.js

```diff
@@ -0,0 +1,8 @@
+export function createInteractiveCreativeFile() {
+  return {
```

Comment:
you could already pass the properties as params in the function

Reply:
Done.